### DISCUSSION TITLE
PNG image format installation

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#5415] [Help]
Getting the below exception for PNG Image

identify: no decode delegate for this image format `PNG' @ error/constitute.c/ReadImage/572.

Help me to add PNG delegates in Linux

**Comment by scottwn** [comment]
I'm having the same problem on opensuse tumbleweed.

I installed libpng-devel and then reinstalled IM, but the issue remains.

**Comment by snibgo** [comment]
@oliverbrown00: please don't spam.
